examples/wallet/package.json

```diff
@@ -89,9 +89,6 @@
     },
     "linux": {
       "target": [
-        "deb",
```

Comment:
why is this?

Reply:
`deb`s and `rpm`s are slower to build and made the workflow of trying out the appimage slower. We might restore them at a later date.